apps/site/assets/css/_schedule-page.scss

```diff
@@ -34,3 +34,56 @@
 .m-schedule-page__sidebar-list {
   padding-left: $base-spacing * 1.5;
 }
+
+.schedule-finder {
+  background-color: $gray-bordered-background;
+  padding: $base-spacing;
+
+  .c-modal__trigger-wrapper {
+    display: inline; // otherwise clicking next to the button fires the event
+  }
+}
+
+.schedule-finder__heading {
+  color: $brand-primary;
+  margin-top: 0;
+
+  &::before {
+    content: '\f073'; // font-awesome calendar
+    font-family: FontAwesome;
+    font-weight: normal;
+    margin-right: $base-spacing / 3;
+  }
+}
+
+.schedule-finder__select-container {
+  background-color: $white;
+  border: solid $border-width $brand-primary-dark;
+  border-radius: $border-radius;
+  margin: $base-spacing / 2 0;
+  padding: $base-spacing / 2;
+  position: relative;
+
+  &.error {
+    border: solid 2px $error-text;
+  }
+}
+
+.schedule-finder__select {
+  appearance: none;
```

Comment:
Does this look OK in IE11? https://caniuse.com/#feat=css-appearance

Reply:
good question!  I'm going to throw it on dev-green and check.